docs/proposals/20210712-sidecarsetimagespullsecrets.md

```diff
@@ -0,0 +1,138 @@
+---
+title: SidecarSetImagePullSecrets
+
+authors:
+- "@veophi"
+  
+reviewers:
+- "@Fei-Guo"
+- "@furykerry"
+- "@FillZpp"
+  
+creation-date: 2021-07-12
+  
+last-updated: 2021-07-12
+  
+status: implementable
+
+---
+# SidecarSetImagePullSecrets
+
+- Provide a way to pull images using `Secret` from private repositories for sidecarSet.
```

Comment:
Please replace all `sidecarSet` to `SidecarSet` or `sidecarset`. The former is kind, and the later is singular. I recommend the former one.
Usually we don't use word like `sidecarSet` except the object variable name in golang.